zer0-os/zOS
# feat(redesign-messenger-chat-container): apply background and update footer positioning

### What does this do?
- applies background and updates footer positioning (messenger chat container)

### Why are we making this change?
- redesign task

### How do I test this?
- run tests as usual
- run UI > check chat container as per below screenshots

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

Before
<img width="1566" alt="Screenshot 2024-09-11 at 11 53 06" src="https://github.com/user-attachments/assets/35a0706f-a7ae-4513-aa49-7a561c361313">

After
<img width="1566" alt="Screenshot 2024-09-11 at 11 52 39" src="https://github.com/user-attachments/assets/81f9f377-b120-45dc-9c90-eda73b3620f0">
